improver/cli/__init__.py

```diff
@@ -458,12 +464,17 @@ def main(prog_name: parameters.pass_name,
     on available command(s).
     """
     args = unbracket(args)
+    local_exec_com = execute_command
```

Comment:
`local_exec_com` -> `exec_cmd`?

Reply:
I can't redefine an outer function, but I can create a local copy and redefine that.

Reply:
Misunderstood what you meant - I'll rename it now :-)